src/basic/generate-af-list.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-cpp -dM -include sys/socket.h - </dev/null | \
+$1 -E -dM -include sys/socket.h - </dev/null | \
```

Comment:
Yeah, but that's a lot of code to support an edge case. It also makes the order of arguments on the command line convoluted. Let me put it this way: if somebody needs to support a compiler with a space in the path, they can provide a patch ;)